spec/data-model/message.json

```diff
@@ -22,27 +22,29 @@
       },
       "required": ["type", "name"]
     },
-    "option": {
-      "type": "object",
-      "properties": {
-        "name": { "type": "string" },
-        "value": {
-          "oneOf": [{ "$ref": "#/$defs/literal" }, { "$ref": "#/$defs/variable" }]
-        }
-      },
-      "required": ["name", "value"]
+    "options": {
```

Comment:
This works, but aesthetically I'd prefer keeping the shallower approach and just duplicating ```"options": { "type": "array", "items": { "$ref": "#/$defs/option" } } ``` (it's nice to have an easily-referenced "option" type).

Reply:
Could you clarify what use case you have in mind for an externally-referrable JSON schema option type?

In my experience (and according to the [reference docs](https://json-schema.org/understanding-json-schema/structuring#defs)), these `$defs` are meant only for internal use within this schema, and that only the top-level entity here should be considered as an external API.

Reply:
As I said, it's just aesthetic. Referencing a singular is both shallower and a bit easier to keep in my human memory when contemplating the schema.